storybook/stories/API/chart/Customized.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {
+  LineChart,
+  XAxis,
+  YAxis,
+  Customized,
+  CartesianGrid,
+  Line,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  Rectangle,
+  Cross,
+} from '../../../../src';
+import { simpleUvPvData } from '../../data';
+
+export default {
+  component: Customized,
+  parameters: {
+    controls: { include: ['data'] },
+  },
+};
+
+const SimpleCustomized = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="80%" height={500}>
```

Comment:
we can't have percentage widths in the embedded storybook it seems

Reply:
Right, please make this a hard 500 for now, until we figure out the issues with relative widths. 
```suggestion
      <ResponsiveContainer width={500} height={500}>
```